package.json

```diff
@@ -42,5 +41,8 @@
     "stylus": "^0.54.5",
     "tape": "^4.9.1",
     "tape-run": "^4.0.0"
+  },
+  "dependencies": {
+    "chart.js": "^2.9.2"
```

Comment:
We do not depend on `tonic` but we do require it in the code.

It would be nice to add a `try { require('tonic') } catch (err) { if (err.ENOFILE) { console.error('good user experience') } }`

Basically in `index.js` for the common use case of someone following the README isntructions or just requiring components, let them know that we do not depend on tonic and it's their job to install tonic and also peer dependencies are broken so we dont use them.

I don't want users of `@optoolco/components` to think that it's a broken module when it throws an exception about a missing dependency and either not using our library or opening a PR to add the missing tonic dependency to the package.json